docs/docs/api-ref.md

```diff
@@ -5,257 +5,3008 @@ layout: docs
 
 <div class="alert alert-info">
     <b>Important:</b> More coming soon! This section is under active construction and might not reflect all the available functionality of the TSC library.
-    Until this reference is completed, we have noted the source files in the TSC library where you can get more information for individual endpoints.
-</div>
+    <!--- Until this reference is completed, we have noted the source files in the TSC library where you can get more information for individual endpoints. --->
```

Comment:
I'll take care of the quick fixes in this PR (against master).  I hear you about the ordering of the API references. I will look into that after I switch to development as base.